src/js/Content/Features/Community/Recommended/FReviewSort.js

```diff
@@ -1,185 +1,175 @@
 import {HTML, Localization, SyncedStorage, TimeUtils} from "../../../../modulesCore";
-import {Background, Feature, Messenger, Sortbox} from "../../../modulesContent";
+import {Background, Feature, Messenger, Sortbox, User} from "../../../modulesContent";
 import {Page} from "../../Page";
 
 export default class FReviewSort extends Feature {
 
+    checkPrerequisites() {
+        return document.querySelectorAll(".review_box").length > 1;
```

Comment:
We only need to check if there're any reviews at all, so the first page will do. I don't think it makes sense to sort if there's only one item, because that's just doing work (if the user has a saved sorting criteria) with no visible change, and is the same logic we applied to other pages we add sorting to. Maybe it makes sense here since we store the review node in the background? Still seems weird it's not doing anything though.

Reply:
Let's say that you have 5 reviews and want to sort by "Helpful".
Further assume that only 4 reviews can be displayed per page and that you're currently on the second / last page with the single review.
With the default "Date" sort, the single review that's displayed to you on page 2 might not be the the least helpful one, because you exit early here.

Reply:
Yep, you're right. I've changed it to check the actual total number of reviews, so even if you're on the last page with a single review, the feature will still apply.